tests/test_utils.py

```diff
@@ -0,0 +1,32 @@
+from owid.catalog.utils import underscore
+
+
+def test_underscore():
+
+    assert underscore("Urban population") == "urban_population"
+    assert (
+        underscore("Urban population (% of total population)")
+        == "urban_population__pct_of_total_population__"
```

Comment:
We should call `name.strip('_')` at the end, i.e. we actually want `urban_population__pct_of_total_population` here and in the cases further down

Reply:
Much better, thanks for an idea.